src/assets/wise5/authoringTool/edit-branch/edit-branch.component.ts

```diff
@@ -215,11 +217,20 @@ export class EditBranchComponent extends AbstractBranchAuthoringComponent {
   protected async submit(): Promise<void> {
     this.submitting = true;
     const params = this.getBranchParams();
+    this.createNewPaths(this.branchPaths, params);
     this.updateTransitions(params);
     this.updateTransitionLogic(params);
     this.saveProject();
   }
 
+  private createNewPaths(branchPaths: any[], params: any): void {
+    branchPaths.forEach((path: any, index: number) => {
```

Comment:
Type CreateBranchParams and get ```this.branchPaths``` from class?
```suggestion
  private createNewPaths(params: CreateBranchParams): void {
    this.branchPaths.forEach((path: any, index: number) => {
```